server/src/utils/getConfig.ts

```diff
@@ -130,6 +135,36 @@ const getPostCallConfig = (defaultConfig: ServerConfigModel): PostCallConfig | u
   return postcallConfig;
 };
 
+const getCosmosDBConfig = (defaultConfig: ServerConfigModel): CosmosDBConfig | undefined => {
+  let cosmosDBConfig: CosmosDBConfig = {
+    dbName: process.env[VV_COSMOS_DB_NAME] ?? (defaultConfig.cosmosDb?.dbName as string)
+  };
+
+  const cosmosDBConnectionString =
+    process.env[VV_COSMOS_DB_CONNECTION_STRING] ?? defaultConfig.cosmosDb?.connectionString;
+  const cosmosDBEndpoint = process.env[VV_COSMOS_DB_ENDPOINT] ?? defaultConfig.cosmosDb?.endpoint;
+
+  if (!(cosmosDBConnectionString || cosmosDBEndpoint)) {
+    return undefined;
+  }
+
+  if (cosmosDBConnectionString) {
+    cosmosDBConfig = {
+      ...cosmosDBConfig,
+      connectionString: cosmosDBConnectionString
+    };
+  }
+
+  if (cosmosDBEndpoint) {
+    cosmosDBConfig = {
+      ...cosmosDBConfig,
+      endpoint: cosmosDBEndpoint
+    };
+  }
+
+  return cosmosDBConfig;
```

Comment:
I think we can make this more concise:

```
const cosmosDBConfig = {
    connectionString: process.env[VV_COSMOS_DB_CONNECTION_STRING] ?? defaultConfig.cosmosDb?.connectionString,
    endpoint: process.env[VV_COSMOS_DB_ENDPOINT] ?? defaultConfig.cosmosDb?.endpoint,
    dbName: process.env[VV_COSMOS_DB_NAME] ?? (defaultConfig.cosmosDb?.dbName as string)
};

if (!(cosmosDBConfig.connectionString || cosmosDBConfig.endpoint)) {
    return undefined;
}

return cosmosDBConfig;
```

Reply:
[bump]

Reply:
Reposting previous comment -

I think we can make this more concise:
```
const cosmosDBConfig = { 
    connectionString: process.env[VV_COSMOS_DB_CONNECTION_STRING] ?? defaultConfig.cosmosDb?.connectionString,
    endpoint: process.env[VV_COSMOS_DB_ENDPOINT] ?? defaultConfig.cosmosDb?.endpoint,
    dbName: process.env[VV_COSMOS_DB_NAME] ?? (defaultConfig.cosmosDb?.dbName as string)
};

if (!(cosmosDBConfig.connectionString || cosmosDBConfig.endpoint)) {
    return undefined;
}

return cosmosDBConfig;
```

Reply:
In this way, if the `endpoint` is undefined, it will result as below:
```javascript
{
  connectionString: 'asdfasfasdf',
  endpoint: undefined,
  dbName: 'testingDBName
}
```

I was trying to avoid to add field that is `undefined`.

Reply:
In that case can we just do:

```
if (cosmosDBConnectionString) {
    cosmosDBConfig.connectionString = cosmosDBConnectionString
}

if (cosmosDBEndpoint) {
    cosmosDBConfig.endpoint = cosmosDBEndpoint
}
```
?

A little nitpicky lol I am just trying to find ways to make the code a little more concise.

Reply:
That would work if we wrap it with if statement. I will update the code.